src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsServiceExtensions.cs

```diff
@@ -46,5 +46,15 @@ public static bool SpansPreprocessorDirective(this ISyntaxFactsService service,
 
         public static bool IsWhitespaceOrEndOfLineTrivia(this ISyntaxFactsService syntaxFacts, SyntaxTrivia trivia)
             => syntaxFacts.IsWhitespaceTrivia(trivia) || syntaxFacts.IsEndOfLineTrivia(trivia);
+
+        public static void GetPartsOfBinaryExpression(this ISyntaxFactsService syntaxFacts, SyntaxNode node, out SyntaxNode left, out SyntaxNode right)
+        {
+            syntaxFacts.GetPartsOfBinaryExpression(node, out left, out _, out right);
+        }
+
+        public static void GetOperatorTokenOfBinaryExpression(this ISyntaxFactsService syntaxFacts, SyntaxNode node, out SyntaxToken token)
+        {
+            syntaxFacts.GetPartsOfBinaryExpression(node, out _, out token, out _);
+        }
```

Comment:
nit: use => for these.

Reply:
@CyrusNajmabadi Shouldn't we just prefer `=>` when the expression fits in the front of member? e.g. `M() => false;` otherwise IMO that's actually harder to read.

Reply:
We've used => for multiline methods for a while now.  It doesn't seem to have a problem. It also helps signify that these methods should try to stay small and not do too much.